frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -44,7 +44,7 @@ function renderReports(reports) {
 
     const collaboratorsWithTags = collaborators.map((collaborator) => (
       <Tag
-        key={collaborator.id}
+        key={collaborator.fullName.slice(1, 3) + collaborator.id}
```

Comment:
Do we need the first three characters of the user's name? If we have dups won't that generate a key that is the same? (since the name and Id will both be the same for duplicated collaborators)